programmers-coding-test/sangjun/문자열내마음대로정렬하기.js

```diff
@@ -0,0 +1,11 @@
+function solution(strings, n) {
+    strings.sort((a,b) => {
+         if (a[n] === b[n]) {
+            return a.localeCompare(b);
+        } else {
+            return a[n].localeCompare(b[n]);
+        }
```

Comment:
들여쓰기 안맞음.
if에서 return 해주므로 else로 감싸줄 필요가 없다.
```suggestion
        if (a[n] === b[n])
            return a.localeCompare(b);
        return a[n].localeCompare(b[n]);
```